assets/src/components/v2/dup/normal_header.tsx

```diff
@@ -30,6 +33,7 @@ const NormalHeader = ({
       showTo={false}
       classModifiers={color}
       accentPattern={accentPattern}
+      playerName={playerName}
```

Comment:
Would it make sense to drop the player name into the existing `version` string, here in the DUP normal header, rather than passing it down to the more generic header component to be added to the string there?

So like
```ts
let version = DUP_VERSION;
if (code) {
  version = `${version}; Maintenance code: ${code}`;
}
if (playerName) {
  version = `${version}-${playerName}`;
}

return <DefaultNormalHeader ... version={version} ... />;
```

Reply:
Now that I look more closely, it might also make sense to add the player name before the maintenance code. (Though we don't actually use those at all in the new DUP app, so this would be more about future-proofing in case we start)

Reply:
I think that makes sense. No reason for `playerName` to exist anywhere in the `DefaultNormalHeader` component.